aspnetcore/fundamentals/logging/index.md

```diff
@@ -624,6 +624,17 @@ Using a third-party framework is similar to using one of the built-in providers:
 
 You can create your own custom providers as well, to support other logging frameworks or your own logging requirements.
 
+## Azure Log Streaming
```

Comment:
Shouldn't that be sentence cased? If it's a product, then shouldn't the following use be upper cased?